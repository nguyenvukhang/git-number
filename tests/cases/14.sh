# big numbers

init 3
_gitnu status
_gitnu add 1
_gitnu add 1000
_gitnu add 1000000
_gitnu add 1000000000
_gitnu add 1000000000000
_gitnu add 1000000000000000
save gitnu status
