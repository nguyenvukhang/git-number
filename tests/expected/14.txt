On branch main

No commits yet

Changes to be committed:
1	[32mnew file:   file_0001[m

Untracked files:
2	[31mfile_0002[m
3	[31mfile_0003[m

